script.js

```diff
@@ -0,0 +1,198 @@
+
+// For all the following exercises create one .js file. Separate the questions using comments.
+
+// Write a console.log statement saying "Hello World!" for each language that you know.
+// For example:
+
+// Halo, dunia! // Indonesian
+
+// Ciao, mondo! // Italian
+
+// Hola, mundo! // Spanish
+
+console.log('Merhaba Dunya');
+console.log('Hello World');
+console.log('Hola, mundo!');
+
+// Consider the following code:
+// console.log('I'm awesome');
+// Copy the code in your .js file and run it. You will see that you will get a SyntaxError. Find a solution for this error. Hint: read the error message carefully, it also gives an indication of where the problem is.
+
+console.log("I'm awesome");
+
+// Declare a variable age and initialize it with an integer, using these exact steps:
+let age;
+
+// First, declare your variable age (do not initialize it yet).
+// Add a console.log statement that explains in words what you think the value of age is
+
+console.log('age is undefined');
+
+// Add a console.log statement that logs the value of age.
+console.log(age);
+
+// Now initialize your variable age with an integer.
+age = 33;
+// Next, add a console.log statement that explains what you think the value of age is.
+console.log('it is 33');
+// Add a console.log statement that logs the value of age.
+console.log(age);
+// Example steps for question 3:
+//   // TODO -> here you declare your variable
+//   console.log('the value of age will be: whateverYouThinkItWillLog');
+//   // TODO -> log the actual value of age
+//   // TODO -> here you initialize your variable
+//   console.log('the value of age will be: whateverYouThinkItWillLog');
+//   // TODO -> log value of age again
+
+// Declare a variable name and assign a string to it.
+let aString = 'world';
+// Write a console.log statement in which you explain in words what you think the value of the string is.
+console.log('it is world');
+// Now console.log the variable name.
+console.log(aString);
+// Now assign a new string to the variable name.
+aString = 'Jupiter';
+// Just like what you did before write a console.log statement that explains in words what you think will be logged to the console.
+console.log('it is Jupiter');
+// Now console.log name again.
+console.log(aString);
+// Example steps for question 4:
+
+//   // TODO -> here you declare AND assign your string
+//   console.log('the value of my string will be: whateverYouThinkItWillLog');
+//   // TODO -> log the actual value of the string to the console
+//   // TODO -> assign a new value to your variable name
+//   console.log('the value of my string will be: whateverYouThinkItWillLog');
+//   // TODO -> log the actual value of the string to the console
+
+// How do you round the number 7.25, to the nearest integer (i.e., whole number)?
+Math.round(7.25);
+console.log(Math.round(7.25));
+// Declare a variable number and assign the number 7.25 to it.
+aNumber = 7.25;
```

Comment:
this is good, but don't forget to use the `let` or `const` keyword for variables.